test-apps/ui-items-providers-test/src/ui-items-providers-test.ts

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { UiItemsManager } from "@itwin/appui-abstract";
+import { SampleTool } from "./tools/SampleTool";
+import { ReducerRegistryInstance } from "@itwin/appui-react";
+// import { SampleContentControl } from "./ui/content/SampleContentControl";
+import { GenericLocateTool } from "./tools/GenericLocateTool";
+import { OpenTraceDialogTool } from "./tools/OpenTraceDialogTool";
+import { IModelApp } from "@itwin/core-frontend";
+import { providerSlice, TestProviderSliceName } from "./store";
+import { GeneralUiItemsProvider } from "./ui/providers/GeneralUiItemsProvider";
+import { OpenAbstractDialogTool } from "./tools/OpenAbstractModalDialogTool";
+import { NetworkTracingFrontstage } from "./ui/frontstages/NetworkTracing";
+import { CustomFrontstage } from "./ui/frontstages/CustomContent";
+
+/** UiItemsProvidersTest is a package that adds some user interface to the iModelApp when its initialize method is called.
+ * Included in the sample are:
+ *   1) `SampleTool` showing how implement a tool with a variety to tool settings items.
```

Comment:
Maybe throw this list up in the README? That may help make it more discoverable what's in the package.